extensions/amp-viafoura/0.1/test/test-amp-viafoura.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2015 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ import {createIframePromise} from '../../../../testing/iframe';
+ require('../../../../build/all/v0/amp-viafoura-0.1.max');
```

Comment:
that's the built file, but normally aren't the tests compiled from the unbuilt files?